### DISCUSSION TITLE
New Mojo (0.5.0) and Modular CLI (0.2.2) release!

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1214] [Announcements]
Hi Everyone!

We just released an update to Mojo 🔥 `v0.5.0` and the Modular 💻 CLI `v0.2.2`

Head over to https://developer.modular.com for instructions to update and install on your platform.

It’s a big update! Make sure to [check out the changelog](https://docs.modular.com/mojo/changelog.html#v0.5.0-2023-11-2)

**Comment by manatlan** [comment]
I've got a [mojo code](https://github.com/manatlan/sudoku_resolver/blob/master/sudoku.mojo) which become extremly slow since the new 0.5.0 version.

- With mojo 0.4.0 : it was 6.65 seconds
- With mojo 0.5.0 : it takes 60 seconds now ?!

?!
ps : is there a way to revert to "0.4.0" ?

**Reply by soraros** [reply]
Maybe you should raise an issue?

**Reply by manatlan** [reply]
done !
https://github.com/modularml/mojo/issues/1216
